packages/Checkbox/stories/examples/CheckboxGrouping.js

```diff
@@ -1,7 +1,12 @@
 import React from "react";
+import styled from "styled-components";
 import { Rule } from "storybook/assets/styles/common.styles";
 import Heading from "@paprika/heading";
-import Checkbox from "../../src/Checkbox";
+import PaprikaCheckbox from "../../src/Checkbox";
+
+const Checkbox = styled(PaprikaCheckbox)`
+  margin-bottom: 16px;
+`;
```

Comment:
This demonstrates how margin would be controlled by a container component. I'm of half a mind to insist we rely on this technique, but I also think that makes it less convenient for consumers.

Reply:
Are you sure about 16px vs 12px that it was before? I think we should only rely on it if it is a direct styling applicable to the directly nested children as is the case here.... But i do think this starts to get a little complicated to follow... Perhaps another alternative could be for the `div` that you removed the `css=` from could also be a styled component?

Reply:
Also, Just wondering about if they were to be displayed `inline` perhaps we could include a `margin-left` in that situation, although i don't think that needs to be done in this pr

Reply:
I made it `16px` so it was more obviously changed (this is just the story, remember).  

The reason I like this approach better than the `css` approach I had before was that we don't need to rely on the `pka-anchor` attribute to select the elements, we just use `styled-components` directly.  But really it's going to be up to the consumer how they do it in the app, this is just a suggestion.

Reply:
I think for horizontal layouts, it'd be better to put the `<Checkbox>` components in a flexbox with columns.  Then the `<Checkbox>` can remain totally agnostic of its environement.